### ISSUE TITLE
`expect` on npm is now mapped to `facebook/jest` not this project

### ISSUE THREAD

**Issue by anglinb** [mjackson/expect#228]
**tl;dr** Following the [installation instructions in the README.md](https://github.com/mjackson/expect#installation) actually installs [facebook/jest](https://github.com/facebook/jest), not this project. 

---

### Explanation 

👋 I recently saw [greenkeeper](greenkeeper.io) create [a PR on a project I contribute to](https://github.com/probot/probot/pull/236) asking us to update our `expect` dependency to `21.0.0`. After investigating, [`expect` on npm is now at version `21.0.0`](https://www.npmjs.com/package/expect) and the link to the source is [github.com/facebook/jest](github.com/facebook/jest), not this project.

@cpojer I saw that you were the last to publish to [https://www.npmjs.com/package/expect](https://www.npmjs.com/package/expect), **did you mean to publish jest instead of this project under the expect name**? 

If so, I think the [README.md Installation section](https://github.com/mjackson/expect#installation) should get changed to match wherever this project is listed on NPM. 

--- 

### Confirming I wasn't crazy

I double checked I wasn't crazy by running  `npm  install --save expect` as the instructions say to do with a fresh node project and I end up with jest installed not this project: 

<img width="696" alt="banners_and_alerts_and_1__brian_brians-macbook-pro____files_endavors_test__zsh_" src="https://user-images.githubusercontent.com/2637602/30038406-00f68922-9179-11e7-9790-0c27cbdf5e2f.png">

Please let me know if I can help sort this out. Thanks 🙌

/cc @mjackson @cpojer

**Comment by cpojer** [comment]
See https://github.com/mjackson/expect/issues/226 and https://github.com/mjackson/expect/pull/227

**Comment by anglinb** [comment]
Missed those--thanks.
